inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp

```diff
@@ -241,17 +241,25 @@ std::map<std::string, std::string> extractMeta(const StageMetaInfo& stageMeta) {
     serializationInfo[ExecGraphInfoSerialization::OUTPUT_LAYOUTS] = layoutStream.str();
 
     std::string outPrecisionsStr;
+    auto fp16name = InferenceEngine::Precision(InferenceEngine::Precision::FP16).name();
+    auto i32name = InferenceEngine::Precision(InferenceEngine::Precision::I32).name();
+    std::string runtimePrecisionStr = i32name;
     ind = 0;
     for (auto &outPrecision : stageMeta.outPrecisions) {
+        // if we have any output precision not equal I32 -> we assume runtimePrecision is FP16
+        auto outPrecisionName = outPrecision.name();
+        if (runtimePrecisionStr != fp16name) {
+            runtimePrecisionStr = (outPrecisionName == i32name) ? outPrecisionName : fp16name;
+        }
```

Comment:
I would have used `InferenceEngine::Precision` instead of `precision.name()`
```suggestion
    auto runtimePrecision = InferenceEngine::Precision::I32;
    ind = 0;
    for (auto &outPrecision : stageMeta.outPrecisions) {
        if (outPrecision != InferenceEngine::I32) {
            runtimePrecision = InferenceEngine::FP16;
        }
```

Reply:
@maxim-kurin  
Good point. Thank you